lib/MusicBrainz/Server/Controller/Role/Load.pm

```diff
@@ -117,15 +117,15 @@ role
 
                 if (defined $pager) {
                     if (!scalar @$lt_groups) {
-                        $c->detach('/error_400');
-                    }
-                    if (scalar @$lt_groups > 1) {
+                        $c->stash->{paged_link_type_group_is_empty} = 1;
```

Comment:
Would it work to just create an empty RelationshipLinkTypeGroup here to stash in paged_link_type_group, then check total_relationships == 0?  That would avoid having to pass a new prop everywhere, at least.

Reply:
Probably. That was my original plan more or less but I wasn't sure whether it would work.

Ideally we should also have a different error (possibly 400 with a message) if the type id selected doesn't exist.

Reply:
Updated this a bit, does it seem better? It does certainly seem simpler, anyway.

Reply:
Thanks!  Looks good now, I think.